test/cli.js

```diff
@@ -588,6 +588,23 @@ describe('cli', function() {
       });
     });
 
+    it('should not error if output directory is a symlink', function(done) {
+      if (process.platform === 'win32') {
+        return done(); // no symlink support on win32
```

Comment:
@xzyfer, Windows has two kinds of symlinks: soft, junction. Both are supported by node.js. The crude difference between Windows and Unix symlinks is instead of a flat file with a simple link, Windows has it as some encrypted hash in registry (a special folder like control panel etc.). Therefore they are not portable (can't zip them, cant git-package them and so on).